### ISSUE TITLE
add Habanos brand icon

### ISSUE THREAD

**Issue by jorgeamadosoria** [simple-icons/simple-icons#6265]
**Brand Name:** several (Cohiba, Montecristo, Romeo y Julieta, Partagás, H. Upmann)
**Website:** http://www.habanos.com/es/unicos-desde-1492/
**Alexa rank:** 379,794

hex `#c9aa4c`

You can assign this one to me. I will take a look at the icons (they are very complex) and create different tickets, one per brand

**Comment by service-paradis** [comment]
Thank you for the request. We will need to asses the popularity for each of these brands. Since they do not have individual websites, which metrics do you propose to evaluate their popularity?

**Comment by jorgeamadosoria** [comment]
> Thank you for the request @jorgeamadosoria. We will need to asses the popularity for each of these brands. Since they do not have individual websites, which metrics do you propose to evaluate their popularity?

I'm sorry, this may have been a bit of cultural arrogance in my part. I apologize.

I guess that I could make a case for the brands chosen, but it's all subjective (smoking is all about personal taste, or lack thereof, I suppose )
For example, we can start with Habanos being the actual word used in the Spanish speaking world to refer to cigars in general. This is a case of generized trademark, like Kleenex for tissues or Band-Aid for bandages.

We have this, hm, "news" report from Business Insider:
https://www.youtube.com/watch?v=vA_j6rjsBec
TL;DR; there are very good Dominican and Nicaraguan cigars, but Cuban cigars are widely considered some of the best, if not straight up the best. The video includes close ups of Cohiba, Romeo y Julieta, Bolivar and Partagas brand cigars, all Habana branded habanos.

We could go by [Cigar Aficionado](https://en.wikipedia.org/wiki/Cigar_Aficionado), a specialized magazine on the matter. If we accept this as an authoritative source (I do), we can see that:
- **Romeo y Julieta** is included in 2020's best 25 cigars: https://www.cigaraficionado.com/top25/2020 (other Habanos brands include Punch, Bolivar and La Aroma de Cuba. I'm not intending to include these as icons, but it's a supporting argument for the overall collection of Habanos products).
- **Partagas** was coincidentally named cigar of the week, with 91/100 points in their evaluation:
![image](https://user-images.githubusercontent.com/15719479/129308219-6abfc72d-2560-4ca5-8298-e84bc9655468.png)
Due to very complex political consequences of the Cuban embargo, Partagas, Montecristo and other smaller brands are produced in both the Dominican Republic and Cuba. Different companies, but the same logo (it's a mess).

- **Cohiba** has several (many) cigars ranked in the magazine, none below 90/100 points. It was the Cigar of the Year in 2010, for example. 
- **Montecristo** was the cigar of the year in 2013. It is also the most widely distributed brand, Both the official [Habanos website](http://www.habanos.com/es/marcas/montecristo/) and Cigar Aficionado.

- Cohiba is also culturally and historically significant (it was established exclusively for the usage of FIdel Castro after 1959, and was only decades later open to the general public). It has a significant cultural status both inside and outside of the US to this day. 

[Altadis](https://en.wikipedia.org/wiki/Altadis), once the biggest producer of cigars in the world (and now a subsidiary of the current biggest producer of cigars, [Imperial Tobacco)](https://en.wikipedia.org/wiki/Imperial_Brands) are / were shareholders of Habanos and also produce the brands for the US market in Dominicana (like I said, the political situation makes all of this complicated). They produce hundreds of millions of this so-called New World cigars specifically as the legally available alternative to Cuban cigars for US consumers. 

FInally, there is a large host of celebrities that are / were consumers of Habanos in some of its variants (this is a bit TMZ-ish, but I trust it gets the point across about popularity and cultural relevance):
- https://thekatynews.com/2020/09/28/7-hollywood-celebrities-and-their-favorite-cigars/ (Arnold, Rihanna, Jay Z and Usher are / were all avid consumers of Cohiba and Montecristo among other Cuban habanos)
- Kennedy famously secured himself a thousand **H.Upmann** cigars hours before enacting the embargo, the stuff of historical legend: https://www.irishcentral.com/roots/history/john-f-kennedy-cigars#:~:text=Kennedy%20managed%20to%20secure%201%2C200,Upmanns%22%20by%20the%20following%20morning.
- https://www.torchcigarbar.com/hollywood-celebrities-and-their-favorite-cigars/ (Robert Downey Jr., Tom Selleck, Jack Nicholson and Pierce Brosnan are / were consumers of Montecristo no.2, Gloria de Cuba and El Rey del Mundo, all Habanos brands)
- https://www.cigaraficionado.com/article/the-top-100-cigar-smokers-of-the-twentieth-century-8111 (MOntecristo, Partagas, Hoyo de Monterrey, Reina Cubana, Cohiba and others make frequent appearances in this list, that includes everyone from Churchill to Clinton). This list also include my personal favorite author, Mark Twain, in a delightfully expected contradictory endorsement to the popularity of Habanos (he is included as smoking 'anything but a Havana' cigar).
- Yet another list of known people and their cigars: https://www.egmcigars.com/blogs/the-cuban-cigars-blog-by-egm-cigars/8-famous-men-who-love-cigars. 

I'm not sure how else we could go around this. The Habanos branded cigars are culturally significant and are managed by the largest company in the world, both in and outside Cuba. They are widely considered by the enthusiasts (I'm not one myself) to be some of the best, if not the best. They consistently rank toward the top in rankings and comparisons. They are sold in every country around the world (literally every country: this is a Habanos store in Pakistan, for example http://www.habanos.com/en/noticias/habanos-especialista-shams-storeen-pakistan/).

Not sure what else to add to make the case for the inclusion of these particular brands. They are not the only ones, but there are over a dozen Habanos brands and I don't think all are similarly popular. These should be more than enough.

**Comment by service-paradis** [comment]
Thank you @jorgeamadosoria for this great explanation!

I just want to clarify that I'm not necessarily against adding every individual brands listed, but we intentionally put a limit on the icons we accept into the project. There are various reason for this (ex. maintenance, icons may need to be updated or removed, package size, ...). This is why we intentionally have clear and comparable measures for this. We're also open to other metrics, which you provide, thank you.

Since it does not use our current metrics to assess popularity, we'll need to discuss this with other @simple-icons/maintainers.

**Comment by jorgeamadosoria** [comment]
> Thank you @jorgeamadosoria for this great explanation!
> 
> I just want to clarify that I'm not necessarily against adding every individual brands listed, but we intentionally put a limit on the icons we accept into the project. There are various reason for this (ex. maintenance, icons may need to be updated or removed, package size, ...). This is why we intentionally have clear and comparable measures for this. We're also open to other metrics, which you provide, thank you.
> 
> Since it does not use our current metrics to assess popularity, we'll need to discuss this with other @simple-icons/maintainers.

I understand. Not everything is going to be easily measured. 
I also understand the need to limit the amount of icons. In this case, that's why I only would like to add 4 brands out of the dozen or so.
Alternatively, we could just add the Habanos logo and that could represent all the brands. It's less aesthetically appealing, but it is more economic.

**Comment by ericcornelissen** [comment]
I don't think we can consider _being ranked highly_ or _celeberties using the product_ or _having historical significance_ as accurate measures of popularity. If you're highly ranked in an obscure thing, you're not popular. Celebrities, almost by definitions, are a small group of people and therefore not a good indication of popularity (arguably their followers would be interested in the brand, making it popular but I'm not convinced that means those people have a need for the logo). Having historical significance seems simply to vague to me to consider, not to mention that it indicates past popularity not present popularity.

based on that I'm hesitant at this point to include these.

**Comment by jorgeamadosoria** [comment]
> I don't think we can consider _being ranked highly_ or _celeberties using the product_ or _having historical significance_ as accurate measures of popularity. If you're highly ranked in an obscure thing, you're not popular. Celebrities, almost by definitions, are a small group of people and therefore not a good indication of popularity (arguably their followers would be interested in the brand, making it popular but I'm not convinced that means those people have a need for the logo). Having historical significance seems simply to vague to me to consider, not to mention that it indicates past popularity not present popularity.
> 
> based on that I'm hesitant at this point to include these.

Well, that's why I included other metrics, like the rankings by Cigar Aficionado and the reach of the stores open, spanning the whole world over.
Also, being the most famous cigar brand in the world is not obscure. It's a multi million dollar business. Granted, it's almost certain that none of the people herein involved smoke, less of all cigars, but this is still hardly an obscure thing to know about.

In any case, I've provided an alternative I can live with: Including the Habanos brand to represent all the cigar brands in one.
That one is Alexa-backed, so it qualifies by default.

**Comment by ericcornelissen** [comment]
> Well, that's why I included other metrics, like the rankings by Cigar Aficionado and the reach of the stores open, spanning the whole world over.

Again, I don't think a ranking (even by an authoritative source) is by itself a reliable indicator of popularity. Being sold worldwide is indeed a decent indication of popularity that I didn't address, not sure I'm convinced it's a good metric but it may be.


> Also, being the most famous cigar brand in the world is not obscure. It's a multi million dollar business.

I didn't intent to say the most famous cigar brand in the world is obscure, I gave an argument for why "being highly ranked" isn't a very reliable indicator of general popularity.


> Granted, it's almost certain that none of the people herein involved smoke, less of all cigars, but this is still hardly an obscure thing to know about.

While that may be true (it is for me at least) that is not a reason to reject it.


> In any case, I've provided an alternative I can live with: Including the Habanos brand to represent all the cigar brands in one.
That one is Alexa-backed, so it qualifies by default.

Habanos can definitely be included, indeed :+1:

**Comment by jorgeamadosoria** [comment]
I've provided the Habanos icon. We can reject all the other ones and review that PR, and put the issue to rest, I think. I remain convinced that the brands can be included, but Habanos is good enough and goes by the guidelines, so we can all move on to more important things

**Comment by PeterShaggyNoble** [comment]
Closing this per @jorgeamadosoria's last comment. If we want to discuss and assess any of the others further then individual issues can be opened for each.
